src/Illuminate/Support/Testing/Fakes/QueueFake.php

```diff
@@ -331,7 +331,7 @@ public function bulk($jobs, $data = '', $queue = null)
     /**
      * Get the connection name for the queue.
      *
-     * @return string
+     * @return void
```

Comment:
why would `getConnectionName()` return `void`?

Reply:
Can we change the method to be abstract instead?

Reply:
that's what it returns. Making it explicit shows you exactly why it is wrong, but on the `Fake` it might be okay.

This is a question whether you want to docblock to help you and show the truth or just be there and lie to you.